saleor/graphql/shop/mutations.py

```diff
@@ -288,3 +288,78 @@ def perform_mutation(cls, _root, info, key_type):
 
         instance.delete()
         return AuthorizationKeyDelete(authorization_key=instance, shop=Shop())
+
+
+class StaffNotificationInput(graphene.InputObjectType):
+    staff_email = graphene.String(
+        required=False,
+        description="Staff email (instead of providing staff member user).",
+    )
+    active = graphene.Boolean(required=False, description="Client secret.")
+    user = graphene.ID(required=False, description="The ID of the user to assign.")
+
+
+class StaffNotificationCreate(ModelMutation):
+    class Arguments:
+        input = StaffNotificationInput(
+            required=True, description="Fields required to update a product."
+        )
+
+    class Meta:
+        description = "Creates a new category."
+        model = account_models.StaffNotification
+        permissions = ("product.manage_products",)
```

Comment:
Descriptions and permissions are wrong

Reply:
I saw that to, fixed already, but good catch :)